react/tic-tac-toe/src/app/screens/Game/components/Square/index.js

```diff
@@ -15,4 +16,9 @@ function Square(props) {
   );
 }
 
+Square.propTypes = {
+  value: PropTypes.string.isRequired,
```

Comment:
square values start as null, so this should not be required, otherwise it will throw a warning